source/adapters/cuda/physical_mem.cpp

```diff
@@ -66,3 +67,31 @@ urPhysicalMemRelease(ur_physical_mem_handle_t hPhysicalMem) {
   }
   return UR_RESULT_SUCCESS;
 }
+
+UR_APIEXPORT ur_result_t UR_APICALL urPhysicalMemGetInfo(
+    ur_physical_mem_handle_t hPhysicalMem, ur_physical_mem_info_t propName,
+    size_t propSize, void *pPropValue, size_t *pPropSizeRet) {
+
+  UrReturnHelper ReturnValue(propSize, pPropValue, pPropSizeRet);
+
+  switch (propName) {
+  case UR_PHYSICAL_MEM_INFO_CONTEXT: {
+    return ReturnValue(hPhysicalMem->getContext());
```

Comment:
Sorry, coming in blind here but I assume we've verified that `hPhysicalMem` isn't `nullptr` by this point? Maybe further up the stack?

Reply:
The validation layer checks these aren't null. It's not always enabled but the assumption adapter implementations make is that non-optional objects are already validated.

Reply:
@frasercrmck happy for this comment to be resolved?